.github/workflows/release.yml

```diff
@@ -47,46 +47,3 @@ jobs:
           version=$(scripts/envDFL.sh version 2>&1 | cut -f1 -d' ')
           zip -r DynaFlowLauncher_V${version}.zip dynaflow-launcher
           curl -H "authorization: Bearer $GITHUB_TOKEN" -H "Content-Type: application/zip" -X POST $(curl -s -H "authorization: Bearer $GITHUB_TOKEN" --request GET https://api.github.com/repos/dynawo/dynaflow-launcher/releases/latest | grep upload_url | cut -d '"' -f 4 | grep -o ".*assets")?name=DynaFlowLauncher_Linux_v${version}.zip --data-binary @DynaFlowLauncher_V${version}.zip
-
```

Comment:
keep this part

Reply:
not sure this part is adapted to release branches? I wasn't there on v1.4 branch

Reply:
yes it is it was just not existing before